browser/src/Services/QuickOpen/RegExFilter.ts

```diff
@@ -39,73 +40,44 @@ export const regexFilter = (
 
     const listOfSearchTerms = searchString.split(" ").filter(x => x)
 
-    let filteredOptions = options
-
-    listOfSearchTerms.map(searchTerm => {
-        filteredOptions = processSearchTerm(searchTerm, filteredOptions, isCaseSensitive)
-    })
-
-    const ret = filteredOptions.map(fo => {
-        const letterCountDictionary = createLetterCountDictionary(searchString)
-
-        const detailHighlights = getHighlightsFromString(
-            fo.detail,
-            letterCountDictionary,
-            isCaseSensitive,
-        )
-        const labelHighlights = getHighlightsFromString(
-            fo.label,
-            letterCountDictionary,
-            isCaseSensitive,
-        )
-
-        return {
-            ...fo,
-            detailHighlights,
-            labelHighlights,
+    // Since the VSCode scorer doesn't deal so well with the spaces,
+    // instead rebuild the term in reverse order.
+    // ie `index browser editor` becomes `browsereditorindex`
+    // This allows the scoring and highlighting to work better.
+    const vsCodeSearchString =
```

Comment:
Thanks for thinking about this! Awesome that the 'refine' functionality will still work 👍